packages/fxa-auth-server/lib/subhub/client.js

```diff
@@ -178,8 +187,41 @@ module.exports = function(log, config) {
   return {
     isStubAPI: false,
 
+    async close() {
+      let promises = [api.close()];
+      if (redis) {
+        promises.push(redis.close());
+      }
+      return await P.all(promises);
+    },
+
     async listPlans() {
-      return api.listPlans();
+      const cacheKey = 'listPlans';
+      const ttlSeconds = config.subhub.plansCacheTtlSeconds;
+      const cacheIsEnabled = ttlSeconds && redis;
```

Comment:
No biggie, but I feel like these should all be defined earlier during initialisation, instead of being method-local. Elsewhere we tend to pull properties off `config` early as a defensive measure, to eliminate any kind of spooky-action-at-a-distance.